lib/faker/default/bank.rb

```diff
@@ -38,18 +60,54 @@ def iban(legacy_country_code = NOT_GIVEN, country_code: 'GB')
         country_code.upcase + iban_checksum(country_code, account) + account
       end
 
+      ##
+      # Produces a random bank name.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Bank.name #=> "ABN AMRO HOARE GOVETT LIMITED"
+      #
+      # @faker.version next
       def name
         fetch('bank.name')
       end
 
+      ##
+      # Produces a random ABA routing number in the format AAAABBBBC.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Bank.routing_number #=> "675861739"
+      #
+      # @faker.version next
       def routing_number
         valid_routing_number
       end
 
+      ##
+      # Produces a random formated routing number in the format XX-XXXX/XXXXX.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Bank.routing_number #=> "675861739"
```

Comment:
This should be `routing_number_with_format`, like so:
```suggestion
      #   Faker::Bank.routing_number_with_format #=> "27-4266/31644"
```